Library/Formula/viewvc.rb

```diff
@@ -0,0 +1,35 @@
+class Viewvc < Formula
+  desc "Browser interface for CVS and Subversion repositories"
+  homepage "http://www.viewvc.org"
+  url "http://viewvc.tigris.org/files/documents/3330/49392/viewvc-1.1.23.tar.gz"
+  sha256 "d14290b23f44e8f5301a7ca9fc946bcd0f8bb8206d9b9c111ac56e19f2588b7d"
+
+  depends_on :python => :optional
+  if build.with? "python"
+    depends_on "subversion" => "with-python"
+  else
+    depends_on "subversion" => ["with-python", "universal"]
```

Comment:
Updated—

> I don't quite get this block. […]

Oops. It was meant to check if the user has Homebrew's python installed or if they're using the System python.

`Viewvc` needs the python bindings for subversion, which don't come installed with the System subversion.

I'd like to just say `depends_on "subversion" => "with-python"`, however [the subversion formula requires `--universal` to be used if you are using the System python](https://github.com/Homebrew/homebrew/blob/c8a8f325d37f68f9d64389a7e2776d66263f301e/Library/Formula/subversion.rb#L81).